web/src/pages/tasks/[taskUrlSlug].tsx

```diff
@@ -149,12 +149,32 @@ export const TaskElement = (props: { task: Task; children?: ReactNode | ReactNod
         <Content>{beforeQuestion}</Content>
         {getPostOnboardingQuestion(props.task)}
         <Content>{afterQuestion}</Content>
+        {props.task.issuingAgency != null || props.task.formName != null ? (
```

Comment:
Since these values can be strings or undefined, you can make this check simpler 
`props.task.issuingAgency || props.task.formName`